css-contain-3/Overview.bs

```diff
@@ -303,6 +303,79 @@ Container Queries</h2>
 		will each respond to their own container context.
 	</div>
 
+	For selectors with pseudo elements, query containers can be established by
+	the <a>shadow-including inclusive ancestors</a> of the <a>ultimate
+	originating element</a>.
+
+	It follows that:
+
+	* Pseudo elements themselves can not be query containers
+	* ''::before'', ''::after'', ''::marker'', and ''::backdrop'' query their
+		originating elements
+	* ''::first-letter'' and ''::first-line'' queries its originating element,
+		even if the <a>fictional tag sequence</a> may push the ''::first-line''
+		past other elements for the purpose of inheritance and rendering
+	* Multiple pseudo elements do not allow pseudo elements to be query
+		containers for other pseudo elements. E.g., the host, but not the part()
+		can be the query container for ''::before'' in ''host::part()::before''
```

Comment:
The `''foo''` syntax is meant for [auto-linking](https://tabatkins.github.io/bikeshed/#autolink-wrappers) 'CSS definition types except property and descriptor'. I'm not sure if that includes the pseudo-elements above, but I don't think it includes example selectors. Probably better to use `<code>` in these cases?

Reply:
It works for the pseudo elements. I've left some as auto-linking, but <code> when it talks about parts of the example selector.